src/pass.h

```diff
@@ -322,9 +327,8 @@ struct PassRunner {
   }
 
   // Add a pass using its name.
-  void add(std::string passName) {
-    doAdd(PassRegistry::get()->createPass(passName));
-  }
+  void add(std::string passName,
+           std::optional<std::string> passArg = std::optional<std::string>());
```

Comment:
```suggestion
           std::optional<std::string> passArg = std::nullopt);
```
This should compile as we use this pattern elsewhere.